bugbug/code_search/searchfox_api.py

```diff
@@ -18,6 +20,20 @@
 
 
 # TODO: we should use commit_hash...
+def get_line_number(elements: Iterable[HtmlElement], position):
+    assert position in ("first", "end")
```

Comment:
Shouldn't it be "start" instead of "first"? In the function call at line 51 you are using "start"

Reply:
Nice catch! Fixed in aabaf245bc291903d21bf3ac06f480fb81b3e3a5.